build-configs/integreat-e2e/assets/app-icon-inverted.svg

```diff
@@ -0,0 +1,19 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="59" height="59" viewBox="0 0 59 59">
```

Comment:
What's up with these new assets? Why are those needed explicitly?

Reply:
Those are basically just a copy of the integreat-test-cms assets, as those are needed to start the live cms app. Maybe you know a solution to reference the assets from the build config?

Reply:
I think symlinking should work, didn't we use that before already?